frontend/src/app/projects/project-overview/project-overview.component.ts

```diff
@@ -38,17 +44,31 @@ import { ProjectWrapperService } from '../service/project.service';
     MatSuffix,
     ReactiveFormsModule,
     FormsModule,
+    MatChipsModule,
   ],
 })
 export class ProjectOverviewComponent implements OnInit {
-  searchControl = new FormControl('');
-  public readonly filteredProjects$ = this.searchControl.valueChanges.pipe(
-    startWith(''),
-    switchMap((query) => this.searchAndSortProjects(query!)),
+  form = new FormGroup({
+    search: new FormControl<string>(''),
+    filter: new FormControl<string | null>(null),
```

Comment:
```suggestion
    projectType: new FormControl<string | null>(null),
```

`filter` is to unspecific if it only accepts project types.